lib/src/chip/chip.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'hc-chip',
+    templateUrl: './chip.component.html',
+    styleUrls: ['./chip.component.scss']
+})
+export class ChipComponent implements OnInit {
+
+    @Input() action: boolean = false;
+    @Input() color: string = 'neutral';
+    chipCursor: string = 'auto';
+    chipDisplay: string = 'inline-block';
+    chipType: string = 'chip';
+
+    constructor() {
+    }
+
+    ngOnInit() {
+        if (this.action) {
+            this.chipCursor = 'pointer';
+            this.chipType += ' close';
+        }
+
+        this.chipType += ' ' + this.color;
```

Comment:
The += will cause an issue when you change this to re-evaluate when the action or color change so make sure you change the way you're handling this